api/src/list/dto/query.dto.ts

```diff
@@ -1,11 +1,16 @@
-import { IsInt, IsOptional, IsString } from 'class-validator';
-import { Type } from 'class-transformer';
+import { IsBoolean, IsInt, IsOptional, IsString } from 'class-validator';
+import { Transform, Type } from 'class-transformer';
 
 export class QueryDto {
   @IsString()
   @IsOptional()
   id?: string;
 
+  @Transform(({ value }) => value === 'true' || value === true || value === 1)
```

Comment:
Yes, despite belief, class-validator does not actually support bools in query strings despite having a IsBoolean function... either that or I couldn't find how to make it work.

Without this we only get back "shared must be a bool".

Reply:
can we use `@Type` like we did here with `per_page`?